plugin/auto/auto_test.go

```diff
@@ -0,0 +1,172 @@
+package auto
+
+import (
+	"context"
+	"strings"
+	"testing"
+
+	"github.com/coredns/coredns/plugin/file"
+	"github.com/coredns/coredns/plugin/pkg/dnstest"
+	"github.com/coredns/coredns/plugin/test"
+
+	"github.com/miekg/dns"
+)
+
+// TODO: consider variables reuse，move to pkg/test?
+var miekAuth = []dns.RR{
```

Comment:
we should not further using `miek.nl` for testing purposes. example.org,com,net is better.

also this looks wholesale copied. What tests are you adding, your commit msg is silent on this?